ui/apps/everest/.e2e/utils/db-cmd-line.ts

```diff
@@ -292,3 +307,64 @@ export const queryTestDB = async (cluster: string, namespace: string) => {
   }
   return result;
 };
+
+export const prepareMongoDBTestDB = async (
+  cluster: string,
+  namespace: string
+) => {
+  await dropTestDB(cluster, namespace);
+
+  // Insert test data into the collection t1 and t2
+  console.log('Preparing test database for MongoDB...');
+  await queryPSMDB(
+    cluster,
+    namespace,
+    'test',
+    'db.t1.insertMany([{ a: 1 }, { a: 2 }, { a: 3 }]);'
+  );
+
+  await queryPSMDB(
+    cluster,
+    namespace,
+    'test',
+    'db.t2.insertMany([{ b: 4 }, { b: 5 }, { b: 6 }]);'
+  );
+
```

Comment:
Can we check that t1 collection is on shard 1 (rs0), and collection t2 is on shard 2 (rs1), just to make sure that both shards have some data before backup and restore.

Reply:
Yeah, I added a new validation for this point!
I also fixed how I was distributing the shards. The collections were not distributing across the shards correctly, so I split them manually and forced them to go to the second shard (rs1). Since rs0 is the main shard, it will retain half of the split in this shard.